CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 #Changelog
 
+## 1.0.24
+### Bugs Fixed
+- Updated to log4j 2.17.1. Resolves two potential vulnerabilities which are only exploitable when using custom log4j configurations that are either writable by untrusted users or log data from the `ThreadContext`.
+
+---
```

Comment:
What happened to change logs for between 1.0.21 and 1.0.24?

Reply:
they weren't created ... and all of them updated log4j versions update as well

Reply:
let me add them.